app.js

```diff
@@ -1,41 +1,30 @@
-var createError = require('http-errors');
-var express = require('express');
-var path = require('path');
-var cookieParser = require('cookie-parser');
-var logger = require('morgan');
-
-var indexRouter = require('./routes/index');
-var usersRouter = require('./routes/users');
-
-var app = express();
-
-// view engine setup
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'pug');
-
-app.use(logger('dev'));
-app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
-app.use(cookieParser());
-app.use(express.static(path.join(__dirname, 'public')));
-
-app.use('/', indexRouter);
-app.use('/users', usersRouter);
-
-// catch 404 and forward to error handler
-app.use(function(req, res, next) {
-  next(createError(404));
+const express = require('express');
+const morgan = require('morgan');
+const cors = require('cors');
+const swaggerUi = require('swagger-ui-express');
+const bodyParser = require('body-parser');
+const { routes } = require('routes');
+
+const app = express();
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(cors({ exposedHeaders: ['access-token', 'expires-in', 'waivio-auth', 'um_session', 'api-key', 'nightmode'] }));
```

Comment:
'waivio-auth', 'um_session', 'api-key', 'nightmode' - if you dont use it, dont write it